content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +200,80 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradable to the `head channel` from the latest version shipped 
+in the minor channel behind. Use skips or skipRange to provide this behaviour. (i.e. using the skipRange in the 
+`stable-3.6` patch for `3.5.<latest>: >= 3.5.<first z of the minor channel>. < 3.6.<new pacth release>`)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-3.5` channel
+- Operator bundles versions from `3.6.0` to `3.6.29` under the `stable-3.6` channel
+
+Then, in this case, your newer patch release `3.6.30` would be configured with `skipRange: >=3.5.25 < 3.6.30` in
+order to only supports upgrading to the newest `3.6.z` from a pretty recent `3.5.z`
+
+**Note** If you would like to ensure that the users of your Operator still able to install the Operator bundle version
+`3.6.29` by using the option `startingCSV` added manually in the subscription then, you will need to also use the `replaces: 3.6.29`
+for this Operator bundle does not get pruned from the index catalog via the OLM resolver.
+
+#### Attention (Be aware of the following example scenario)
+
+If you have the channel `stable-3.7` where the head of channel is `3.7.10` and then, you provide a new patch release
+with a bug fix using the Operator bundle version `3.7.11` then, imagine that you configure your new publication to skip all
+that was published under the channel `stable-3.7` (e.g. skipRange: `">= 3.7.0 < 3.7.11"` replaces: `3.6.z <latest release on 3.6 channel>` ).
+
+You could ensure that your users can move forward easier to the latest version since in this range `">= 3.7.0 < 3.7.11"`
+as they will be able to install the new `3.7.11` release from  `3.6.z <latest release on 3.6 channel>` directly 
+instead of get installed first the Operator version `3.7.0` which we know that have bug fixes in place already.
+
+However, **attention** that means you would no longer able to provide any patch release to the minor channel `stable-3.6`.    
+Note that if you publish `3.6.latest+1` when this version comes out, and your users upgrades to it, they will have no way to
+upgrade from `3.6.latest+1` to any solution published under the channel `stable-3.7`.
+
+## Recommended upgrade path cross-channel
+
+After subscribing to a channel, OLM users are able to update their [Subscription][subscription] to
+consume the solutions from another channel, for example, by moving from the fast channel to
+the stable one.
+
+Because of this, it is recommended configure your bundles versions to allow cross-channel updates at least 
+between two consecutive major/minor versions (e.g. by having installed the head of the channel `stable-2.9` 
+your users can get installed the first version under `stable-2.10`)
```

Comment:
this whole section seems redundant to the previous section(` Recommended upgrade path for patch release (bug fixes)`) and the recommendation is worse (because it recommends upgrade paths which take the user through the X.Y.0 version of a minor stream, instead of directly to X.Y.latest).

everything recommended in the previous section already allows for a user to move from "stable-2.9" to "stable-2.10" if it is done correctly.  (i.e. the previous section covers both patch upgrades *and* minor version upgrades).

I suggest dropping this section and simply retitling the previous section to clarify that it's upgrade graph structure recommendations generally.